CHANGES/1418.bugfix.rst

```diff
@@ -0,0 +1 @@
+fix TelegramObject from DefaultBotProperties is not JSON serializable
```

Comment:
Should starts with capitalized word.
Also this line can be not understandable for end-users.